admin/services/prepare-check.service.js

```diff
@@ -61,10 +66,14 @@ const service = {
   }
 }
 
-function buildLookupKey (checkCode) {
+function buildPreparedCheckLookupKey (checkCode) {
```

Comment:
There is a `redisKeyService` that seems like a good fit for this code.

Reply:
ah yes, thanks for the reminder 👍